src/Surfnet/ServiceProviderDashboard/Application/ViewObject/EntityActions.php

```diff
@@ -86,8 +86,7 @@ public function getEnvironment()
     public function allowEditAction(): bool
     {
         $notEditable =
-            $this->isPublishedToProduction()
-            || $this->readOnly
+            $this->readOnly
```

Comment:
Random question: Is this change tested?

Reply:
Albeit a small change of an existing test, yes it was covered in test here: `tests/integration/Application/ViewObject/EntityActionsTest.php:144`

Reply:
:100: